analyzer/src/main/resources/scripts/init.gradle

```diff
@@ -353,6 +351,22 @@ class AbstractDependencyTreePlugin<T> implements Plugin<T> {
             }
         }
 
+        private static boolean canBeResolved(org.gradle.api.artifacts.Configuration configuration) {
+            // The DeprecatableConfiguration class is available since Gradle 6.0.
+            boolean isDeprecatedForResolving = false
+            if (configuration.metaClass.respondsTo(configuration, "getResolutionAlternatives")) {
+                isDeprecatedForResolving = configuration."getResolutionAlternatives"() != null
```

Comment:
Nit: Maybe use single quotes like in `hasProperty('canBeResolved')`?

Reply:
Also, just wondering, are the quotes in the second line even required? Line 364 also seems to require no quotes after the check that the property is present.

Reply:
Also semantically, I would have preferred to check whether `configuration` is a `DeprecatableConfiguration`, and if so call `getResolutionAlternatives()`. That would make more clear, why you're referring to that class in the comment.

Reply:
I have adapted the comment to make clear why I check for the existence of the function.

Reply:
> I have adapted the comment to make clear why I check for the existence of the function.

Did that happen yet?

Reply:
Ah, sorry, I was looking for this in the commit message, not in the code comment.